internal/testcafe/config.go

```diff
@@ -105,7 +105,15 @@ func FromFile(cfgPath string) (Project, error) {
 			env[k] = os.ExpandEnv(v)
 		}
 		p.Suites[i].Env = env
+		setDefaultValues(&p.Suites[i])
 	}
-
 	return p, nil
 }
+
+func setDefaultValues(suite *Suite) {
+	// If value is 0, it's assumed that the value has not been filled.
+	// So we define it to the default value: 1 (full speed).
+	if suite.Speed < .01 {
```

Comment:
So why not `suite.Speed == 0` or `suite.Speed < 1`?

Reply:
@alexplischke  Expected value from TestCafe is between `0.01` and `1`.
This was made to avoid invalid values (0 and negative). I should add checks for `suite.Speed` > 1 to completely check it.

Reply:
Ah! Would be good to add that info here 💡